e2e/tests/session_steps_test.go

```diff
@@ -213,6 +217,57 @@ func (suite *suite) TheModuleParametersAreSetAsFollows(moduleName string, params
 	suite.TheAccountSendsAnAuthzExecMessageToUpdateAllModuleParams("pnf", moduleName, params)
 }
 
+func (s *suite) TheApplicationEstablishesAWebsocketsConnectionForService(appName string, testWsServiceId string) {
+	app, ok := accNameToAppMap[appName]
+	require.True(s, ok, "application %s not found", appName)
+
+	sharedParams := s.getSharedParams()
+
+	currentHeight := s.getCurrentBlockHeight()
+	nextSessionStartHeight := sharedtypes.GetNextSessionStartHeight(&sharedParams, currentHeight)
+	s.waitForBlockHeight(nextSessionStartHeight)
+
+	s.wsCloseHeight = sharedtypes.GetClaimWindowOpenHeight(&sharedParams, nextSessionStartHeight) - 1
+
+	header := http.Header{}
+	header.Add("App-Address", app.Address)
+	header.Add("Target-Service-Id", testWsServiceId)
+
+	wsUrl, err := url.Parse(pathUrl)
+	require.NoError(s, err)
+
+	wsUrl.Scheme = "ws"
+
+	wsConn, _, err := websocket.DefaultDialer.Dial(wsUrl.String(), header)
+	require.NoError(s, err)
+
+	subscriptionMsg := `{"id":1,"jsonrpc":"2.0","method":"eth_subscribe","params":["newHeads"]}`
+	err = wsConn.WriteMessage(websocket.TextMessage, []byte(subscriptionMsg))
+	require.NoError(s, err)
+
+	s.wsConn = wsConn
+}
+
+func (s *suite) TheUserReceivesEthereumSubscriptionEvents() {
+	ctx := s.readEthSubscriptionEvents()
```

Comment:
Ditto: Eth or EVM?

I think it should be the latter because theoretically these helpers WILL support all EVM chains.